spec/src/main/asciidoc/tracing.adoc

```diff
@@ -169,275 +168,8 @@ Calling the OpenTelemetry API directly MUST work in the same way and yield the s
-
-| `otel.exporter.otlp.traces.compression`
-| The compression type to use on OTLP trace requests.
-Options include gzip.
-By default no compression will be used.
-
-| `otel.exporter.otlp.timeout`
-| The maximum waiting time, in milliseconds, allowed to send each OTLP trace, metric, and log batch.
-
-Default value: `10000`
-
-| `otel.exporter.otlp.traces.timeout`
-| The maximum waiting time, in milliseconds, allowed to send each OTLP trace batch.
-
-Default value: `10000`
-|===
-
-If Environment Config Source is enabled for MicroProfile Config, then the environment variables as described by the OpenTelemetry SDK Autoconfigure are also supported.
-
-[[sec:service-loader-support]]
-==== Service Loader Support
-Implementation will load additional configuration related components by means of service loader.
-This allows the application to define its own metadata and trace handling behavior.
-The following components are supported
-
-[options=header]
-|===
-| Component interface | Purpose
-
-| https://javadoc.io/doc/io.opentelemetry/opentelemetry-sdk-extension-autoconfigure-spi/{otel-java-version}/io/opentelemetry/sdk/autoconfigure/spi/ConfigurablePropagatorProvider.html[`ConfigurablePropagatorProvider`]
-| Provides implementation for a name referred in `otel.propagators`
-
-| https://javadoc.io/doc/io.opentelemetry/opentelemetry-sdk-extension-autoconfigure-spi/{otel-java-version}/io/opentelemetry/sdk/autoconfigure/spi/traces/ConfigurableSpanExporterProvider.html[`ConfigurableSpanExporterProvider`]
-| Provides implementation for a name referred in `otel.traces.exporter`
-
-| https://javadoc.io/doc/io.opentelemetry/opentelemetry-sdk-extension-autoconfigure-spi/{otel-java-version}/io/opentelemetry/sdk/autoconfigure/spi/traces/ConfigurableSamplerProvider.html[`ConfigurableSamplerProvider`]
-| Provides implementation for a name referred in `otel.traces.sampler`
-
-| https://javadoc.io/doc/io.opentelemetry/opentelemetry-sdk-extension-autoconfigure-spi/{otel-java-version}/io/opentelemetry/sdk/autoconfigure/spi/AutoConfigurationCustomizerProvider.html[`AutoConfigurationCustomizerProvider`]
-| Customizes configuration properties before they are applied to the SDK
-
-| https://javadoc.io/doc/io.opentelemetry/opentelemetry-sdk-extension-autoconfigure-spi/{otel-java-version}/io/opentelemetry/sdk/autoconfigure/spi/ResourceProvider.html[`ResourceProvider`]
-| Defines resource attributes describing the application
-|===
-
-Behavior when multiple implementations are found for a given component name is undefined.
-Behavior when customizer changes other properties than those listed in the spec is also undefined.
-
 [[sec:semantic-conventions]]
-=== Semantic Conventions
+=== Trace Semantic Conventions
 The https://github.com/open-telemetry/opentelemetry-java/tree/v{otel-java-version}/semconv/src/main/java/io/opentelemetry/semconv[Trace Semantic Conventions] for Spans and Attributes MUST be followed by any compatible implementation.
```

Comment:
Semantic conventions in java have been moved here: https://github.com/open-telemetry/semantic-conventions-java

Reply:
Moving to stable conventions should be a separate PR.